### ISSUE TITLE
CheckStyle-IDEA (5.74.0) exception in JetBrains IntelliJ 2024.2

### ISSUE THREAD

**Issue by jshiell** [jshiell/checkstyle-idea#647]
```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'org.infernus.idea.checkstyle.actions.ScanCurrentFile' must override `getActionUpdateThread()` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: CheckStyle-IDEA]
                at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
                at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
                at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
                at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
                at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:201)
 ```
 
IntelliJ was upgraded overnight. Starting getting errors including this.

on https://plugins.jetbrains.com/plugin/1065-checkstyle-idea see there are newer versions of the plug-in but not sure how to upgrade the plugin within the IntelliJ IDE.

![image001](https://github.com/user-attachments/assets/2afbcdc5-c439-4cdc-868f-ed7b3835cbda)

(Via email)

**Comment by jshiell** [comment]
Probably a dup of #634.

**Comment by jshiell** [comment]
Looks like it is, and fixed as of 5.88.1.
